code/datums/components/engraved.dm

```diff
@@ -0,0 +1,53 @@
+/**
+ * # engraved component!
+ *
+ * component for walls that applies an engraved overlay and lets you examine it to read a story (+ art element yay)
```

Comment:
can this be not-wall only, also if its wall only why is it a component

Reply:
I just wanna say that this will definitely be genericized but do not bash something being a component even if it only works for one type, components are great even then because of their consolidated behavior to one file